import-export-cli/integration/README.md

```diff
@@ -53,8 +53,9 @@ apictl-version: 4.0.1
 ```   
 
 
-3. Build the apictl source to create the archive distribution of your choice.
-
+4. Build the apictl source to create the archive distribution of your choice.
+   > **Note:** (Optional) Delete API-CTL user directories (`rm ~/.wso2apictl.local ~/.wso2apictl`)
```

Comment:
Shall we remove this "Optional" word from here? I think it is better to remove/backup the existing dirs always before running tests.  @uvindra WDYT?

Reply:
I also felt the same, after sometime the PR sent :) fixed with https://github.com/wso2/product-apim-tooling/pull/748/commits/b5d1e8d8ab5d80bd0321b53cda71dd81c541dbbb